grafana-plugin/.config/webpack/webpack.config.ts

```diff
@@ -95,14 +95,6 @@ const config = async (env): Promise<Configuration> => {
             },
           },
         },
-        {
-          test: /\.css$/,
-          use: ['style-loader', 'css-loader'],
-        },
-        {
-          test: /\.s[ac]ss$/,
-          use: ['style-loader', 'css-loader', 'sass-loader'],
-        },
```

Comment:
We shouldn't edit this file ideally, it is scaffolded. Instead we should override those rules from within our webpack config. I can see we use Merge strategy for module rules, maybe we can try out Replace?

Reply:
Discussed that it is temp change so we can leave it for now